EIPS/fundable-token-draft.md

```diff
@@ -0,0 +1,269 @@
+
+| Parameter | Description |
+| ---------|-------------|
+| operationId |he unique ID per token holder to identify the request |
+| walletToFund | The wallet to be funded on behalf.
+| value | The amount to be funded. |
+| instruction | A string including the payment instruction. |
+
+#### cancelFund
+
+Cancels a funding request.
+
+| Parameter | Description |
+| ---------|-------------|
+| orderer | The address of the orderer, to correlate the right data.
+| operationId | The unique ID per token holder to identify the request that is going to be cancelled. This can only be done by token holder, or the fund initiator. |
+| reason | The unique ID per token holder to identify the request that is going to be cancelled. This can only be done by token holder, or the fund initiator. |
+
+
+#### processFund
+
+Marks a funding request as on process. After the status is on process, order cannot be cancelled.
+
+| Parameter | Description |
+| ---------|-------------|
+| orderer | The address of the orderer, to correlate the right data.
+| operationId | The unique ID per orderer to identify the request is in process.
+
+#### executeFund
+
+Issues the amount of tokens and marks a funding request as executed.
+
+| Parameter | Description |
+| ---------|-------------|
+| orderer | The address of the orderer, to correlate the right data.
+| operationId | The unique ID per orderer to identify the request that has been executed.
+
+#### rejectFund
+
+Rejects a given operation with a reason.
+
+| Parameter | Description |
+| ---------|-------------|
+| orderer | The address of the orderer, to correlate the right data.
+| operationId | The unique ID per orderer to identify the request that has been executed.
+| reason | The specific reason that explains why the fund request was rejected. EIP 1066 codes can be used |
+
+#### isApprovedToOrderFund
+
+Checks that given player is allowed to order fund requests, for a given wallet.
+
+| Parameter | Description |
+| ---------|-------------|
+| walletToFund | The wallet to be funded, and checked for approval permission.
+| orderer | The address of the orderer, to be checked for approval permission.
+
+
+#### retrieveFundData
+
+Retrieves all the fund request data. Only operator, tokenHolder, and orderer can get the given operation data.
+
+| Parameter | Description |
+| ---------|-------------|
+| orderer | The address of the orderer, to correlate the right data.
+| operationId | The unique ID per token holder to identify the fund order.
+
+
+## Events
+
+#### FundOrdered
+
+Emitted when an token wallet owner orders a funding request.
+
+| operationId | The unique ID per token holder to identify the request |
+| walletToFund | The wallet that the player is allowed to start funding requests |
+| value | The amount to be funded. |
+| instruction | A string including the payment instruction. |
+
+#### FundInProcess
+
+Emitted when an operator accepts a funding request, and the operation is in process.
```

Comment:
`accepts` sounds to me, as if the funding request is already executed. Maybe we can use `receives` or `starts to proceess`